code/modules/clothing/clothing.dm

```diff
@@ -655,23 +655,6 @@ BLIND	 // can't see anything
 	if (!is_mining_level(T.z))
 		return . * high_pressure_multiplier
 
-/obj/item/clothing/get_examine_line(skip_examine_link = FALSE)
```

Comment:
Why is the new code being gutted? 

How does the new examine() render look when accessories are at their max, compared to what I set it to render?

Reply:
Because I already added it, you can see in the above image that it appears

Reply:
The mob examine code is still there

Reply:
<img width="1582" height="317" alt="image" src="https://github.com/user-attachments/assets/3aa06fcf-2215-4a2b-8531-074cb8f0cbda" />